lib/bgs/service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module BGS
+  class Service
+    # include BGS::Exceptions::BGSErrors
```

Comment:
Can this comment be removed?

Reply:
I've removed this comment, thank you for pointing that out.